.github/workflows/platform-build.yml

```diff
@@ -1,4 +1,5 @@
 name: Platform packages build and deploy to -develop/
+run-name: Build ${{ inputs.dry-run == true && 'w/o deploy' || '& deploy' }}${{ inputs.overwrite == true && '(+overwrite)' || '' }} to dist-${{inputs.stack}}-develop/ ${{ inputs.publish == false && 'w/o publishing packages into Composer platform repository' || '' }}
```

Comment:
:wow: What a 1-liner. It would be nice if this could have some room to breathe in a script or something, but I know this is a limitation of actions.